api/v1alpha1/nonadminbackup_types.go

```diff
@@ -32,14 +47,21 @@ type NonAdminBackupSpec struct {
 
 	// BackupSpec defines the specification for a Velero backup.
 	BackupSpec *velerov1api.BackupSpec `json:"backupSpec,omitempty"`
-
-	// BackupStatus captures the current status of a Velero backup.
-	BackupStatus *velerov1api.BackupStatus `json:"backupStatus,omitempty"`
 }
 
 // NonAdminBackupStatus defines the observed state of NonAdminBackup
 type NonAdminBackupStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// +optional
+	Phase NonAdminBackupPhase `json:"phase,omitempty"`
```

Comment:
The `Phase` and `FailureReason` might be redundant here, becasue conditions consists of similar fields like `Conditions.Type` , `Conditions.Reason` and `Conditions.Message` 
Reference: https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Condition